src/fiatconnect/saga.ts

```diff
@@ -189,27 +189,35 @@ export function* handleSubmitFiatAccount({
             yield put(submitFiatAccountKycApproved())
             yield delay(500) // Allow user to admire green checkmark
             break
-          // Denied, Expired, and Pending all fall through to the default case.
```

Comment:
we were right about how default was working right? and the problem was just that an unrecognized status was cropping up somehow?

Reply:
the issue is that the cases were not falling through _directly_ to default, they were falling through to _subsequent_ KYC status cases _first_ calling `navigate` multiple times, and only _then_ to default. The change to the default case is to bring this in line with the _other_ KYC status switch/case we have, and it allows us to identify when we receive an unexpected KYC status.